spec/requests/providers_spec.rb

```diff
@@ -500,7 +500,7 @@ def have_endpoint_attributes(expected_hash)
     it 'allows provider specific attributes to be specified' do
       allow(ManageIQ::Providers::Azure::CloudManager).to receive(:api_allowed_attributes).and_return(%w(azure_tenant_id))
       tenant = FactoryBot.create(:cloud_tenant)
-      api_basic_authorize collection_action_identifier(:providers, :create)
+      api_basic_authorize "ems_cloud_new"
```

Comment:
let's add a helper in spec/support/api/helpers.rb so we don't hard code these around and sprouting dups over time.
Maybe something like:
```
collection_action_classed_identifier(type, action, method, klass)
```

with this, above can be replaced with:

```
api_basic_authorize collection_action_classed_identifier(:providers, :create, :post, "ManageIQ::Providers::CloudManager")
```

Reply:
@abellotti any chance that could be follow-up? I'd like to close the bug and then refactor if you're okay with that.

Reply:
Juuuuuust kidding.

Reply:
Thanks @d-m-u for adding the helper and updating the specs. 👍